src/main/java/seedu/duke/Task.java

```diff
@@ -0,0 +1,79 @@
+package seedu.duke;
+
+/**
+ * Handles the tasks relating to the ToDoList
+ */
+public class Task {
+
+    private int id;
+    private String task;
+    private boolean done;
```

Comment:
Should this be renamed to `isDone`?
```suggestion
    private boolean isDone;
```

Reply:
I agree, `isDone` makes more sense in this case. Thank you for the suggestion!